publish_unit_test_results.py

```diff
@@ -832,6 +839,8 @@ def check_var(var: str, name: str, label: str, allowed_values: Optional[List[str
     report_individual_runs = get_var('REPORT_INDIVIDUAL_RUNS') == 'true'
     dedup_classes_by_file_name = get_var('DEDUPLICATE_CLASSES_BY_FILE_NAME') == 'true'
     hide_comment_mode = get_var('HIDE_COMMENTS') or 'all but latest'
+    # Comment on PRs if COMMENT_ON_PR is not set to 'false'
```

Comment:
I think this comment does not add much value, no extra information the code is not providing itself

Reply:
I added it after I messed up this line yesterday. Looking at it now, I think it's quite self-explanatory.